module2/task2.py

```diff
@@ -0,0 +1,32 @@
+import re
+import socket
+
+
+def check_ip_by_re(ip_address):
+    if re.match("([0-9]{1,3}[\.]){3}[0-9]{1,3}", str(ip_address)):
+        return True
+    return False
```

Comment:
You could also just return a value of "if" statement and it's better way to do it in Python

return re.match("([0-9]{1,3}[\.]){3}[0-9]{1,3}", str(ip_address))

Reply:
Oops, got it